src/codewhisperer/util/commonUtil.ts

```diff
@@ -25,7 +25,7 @@ export function asyncCallWithTimeout<T>(asyncPromise: Promise<T>, message: strin
 }
 
 export function isInlineCompletionEnabled() {
-    return semver.gte(vscode.version, '1.68.0') && getInlineSuggestEnabled() && !isCloud9()
+    return getInlineSuggestEnabled() && !isCloud9()
```

Comment:
This change was already possible since "engines" was already updated: https://github.com/aws/aws-toolkit-vscode/blob/183ebece93b0ed75a9f074f24a2c5d19e2fbcff4/package.json#L15-L16